### PR TITLE
Adding "remake.yml" file.

### PR DESCRIPTION
**Pull request by RAtshan**
Created a "remakefile" branch, then created file called "remake.yml".

### CODE REVIEW

**Comment by RAtshan** [comment]
Build message screenshot: 
image

**Comment by aappling-usgs** [comment]
Hi Rasha, I think the code I'm seeing may not be consistent with what I remember from Friday or from the screenshot you shared above. Could you double check that all your changes have been committed and pushed?

**Review by RAtshan — COMMENTED** [review]
Build message screenshot:

image

**Review by aappling-usgs — CHANGES_REQUESTED** [review]
Hooray, how exciting to have a pipeline that runs without error!

Your next task is to rework the function arguments a bit. Our goal is to capture within *remake.yml* all of the dependencies and products (data inputs and outputs) of each function. So your `fetch_data` call is already just fine: there are no data inputs, and the one product is the creation of *1_fetch/out/model_RMSEs.csv*. `eval_data` is also OK because there's just one data input (the *model_RMSEs.csv* file and the only product is the object that gets produced and assigned to `eval_data`.

`plot_results` could use some more work. Rather than accepting an output **directory**, change this function and line 25 so that the second argument is a complete file path equal to the target name on line 24. In this way you'll be specifying all the inputs (`eval_data`) and all the outputs (`3_visualize/out/figure_1.png`) within the `command` for that recipe.

I'm also ready to see a more accurate argument name for the first argument to `plot_results`. You're not passing in a filename anymore, so where `mendota_file` made sense for the calls to `fetch_data` and `load_data`, here there should be some different argument name (that doesn't say "file") for the evaluation data table that you pass in.

Push up your changes and share another screenshot when you have these changes working.

**Review by RAtshan — COMMENTED** [review]
Build message screenshot:


image

File: 3_visualize/src/visualize_data.R [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ############## Visualize data  ######################
 
 
-plot_results <- function(mendota_file, project_output_dir) {
-              
-                    png(file = file.path(project_output_dir, 'figure_1.png'), width = 8, height = 10, res = 200, units = 'in')
+plot_results <- function(mendota_proc_data, project_output_dir) {
+ 
+                    png(mendota_proc_data, project_output_dir = file.path(project_output_dir, names = 'figure_1.png'), width = 8, height = 10, res = 200, units = 'in')
```
Comment (aappling-usgs) [inline 3_visualize/src/visualize_data.R]: this looks wrong (and not like what i remember from our most recent phone call) - check again for updates to commit?

**Review by aappling-usgs — CHANGES_REQUESTED** [review]
Mostly looks great, but did you commit your most recent version of visualize_data.R?

**Review by aappling-usgs — APPROVED** [review]
You did it!! Nice work, Rasha!

**Pull request merged**